dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentletLoader.java

```diff
@@ -0,0 +1,969 @@
+        sb.append("#set($ContentLastModDate= $date.toDate(\"yyyy-MM-dd H:mm:ss\", \"")
+            .append(modDateStr)
+            .append("\"))")
+            .append("#set($ContentLastModUserId= \"")
+            .append(content.getModUser())
+            .append("\")");
+        if (content.getOwner() != null)
+            sb.append("#set($ContentOwnerId= \"")
+                .append(content.getOwner())
+                .append("\")");
+
+        // Structure fields
+
+        ContentType type = content.getContentType();
+
+        List<Field> fields = type.fields();
+
+
+        String widgetCode = "";
+
+        for (Field field : fields) {
+
+            String contField = field.dbColumn();
+            String contFieldValue = null;
+            Object contFieldValueObject = null;
+            String velPath = (!EDIT_MODE) ? "live/" : "working/";
+            if (field instanceof HiddenField || field instanceof ConstantField) {
+                if (field.variable()
+                    .equals("widgetPreexecute")) {
+                    continue;
+                }
+                if (field.variable()
+                    .equals("widgetCode")) {
+                    widgetCode = "#set($" + field.variable() + "=$velutil.mergeTemplate(\"" + velPath + content.getInode()
+                            + File.separator + field.inode() + "." + VelocityType.FIELD.fileExtension + "\"))";
+                    continue;
+                } else {
+                    String fieldValues = field.values() == null ? "" : field.values();
+                    if (fieldValues.contains("$") || fieldValues.contains("#")) {
+                        sb.append("#set($")
+                            .append(field.variable())
+                            .append("= $velutil.mergeTemplate(\"")
+                            .append(velPath)
+                            .append(content.getInode())
+                            .append(File.separator)
+                            .append(field.inode())
+                            .append(".")
+                            .append(VelocityType.FIELD.fileExtension)
+                            .append("\"))");
+                    } else {
+                        sb.append("#set($")
+                            .append(field.variable())
+                            .append("= \"")
+                            .append(UtilMethods.espaceForVelocity(fieldValues)
+                                .trim())
+                            .append("\")");
+                    }
+                    continue;
+                }
+            }
+
+
+            if (UtilMethods.isSet(contField)) {
+                try {
+                    contFieldValueObject = conAPI.getFieldValue(content, field);
+                    contFieldValue = contFieldValueObject == null ? "" : contFieldValueObject.toString();
+                } catch (Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)